Lib/fontTools/pens/roundingPen.py

```diff
@@ -8,7 +8,9 @@
 
 class RoundingPen(FilterPen):
     """
-    Filter pen that rounds point coordinates and component XY offsets to integer.
+    Filter pen that rounds point coordinates and component XY offsets to integer. For
+    rounding the component scale values, a separate round function can be passed to the
```

Comment:
technically it's not just "scale" values (but also shear or rotation anything that 2x2 matrix can do), i'd just call it "component transform" values